addons/survey/models/survey_survey.py

```diff
@@ -583,9 +583,33 @@ def _get_pages_or_questions(self, user_input):
             else:
                 result = self.question_and_page_ids.filtered(
                     lambda question: not question.is_page or not is_html_empty(question.description))
+                result = self._get_questions_except_invalid_conditionals(result)
 
         return result
 
+    def _get_questions_except_invalid_conditionals(self, questions):
+        """ This method takes a survey.question recordset and
+            ignores conditional questions that have an invalid trigger. """
+
+        invalid_questions = self.env['survey.question']
+        for question in questions:
+            if not question.is_conditional:
+                continue
+            if (question.triggering_question_id in invalid_questions
+               or not self._is_valid_conditional_question(question)):
+                invalid_questions |= question
```

Comment:
Maybe simple enough to be back into one function?

```suggestion
        for question in questions.filtered(lambda q: q.is_conditional):
            trigger = question.triggering_question_id
            if (
                    trigger in invalid_questions
                    or trigger.is_page
                    or trigger.question_type not in ['simple_choice', 'multiple_choice']
                    or ...):
               excluded_questions |= question
```